README.md

```diff
@@ -13,7 +14,8 @@ an error panel will open detailing the LaTeX error.
 ## Requirements
 
 * JupyterLab 0.31
-* An application that can compile `.tex` files to PDF on your notebook server (e.g., `pdflatex` or `xelatex`).
+* An application that can compile `.tex` files to PDF (e.g., `pdflatex`, `xelatex`, or `pdflatex.exe` on Windows with MiKTeX). This application must be available as a command in the same environment as the notebook server.
```

Comment:
This now sounds like `pdflatex` and `xelatex`are only commands available on Windows with MikTeX. I'd separate out this into two parts with a semicolon.